### ISSUE TITLE
tf.contrib.nn.alpha_dropout returns tensor with no shape

### ISSUE THREAD

**Issue by tocab** [tensorflow/tensorflow#25216]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): y
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): win10
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.12.0
- Python version: 3.6.7

**Describe the current behavior**
I tried out selu activation and also used alpha_dropout for that. When wrapping tf.contrib.nn.alpha_dropout() around my dense layer, a Tensor with no shape will be returned and therefore I can't feed the next layer with this tensor.

A solution for this can be to assign the input shape to the returned tensor, like in the tf.nn.dropout() Layer.

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
